engine/src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -1,75 +1,64 @@
 package cromwell.server
 
-import java.util.concurrent.TimeoutException
+import akka.actor.{ActorContext, Props}
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.Directives._
 
-import akka.actor.{ActorContext, ActorSystem, Props}
 import akka.stream.ActorMaterializer
-import com.typesafe.config.Config
 import cromwell.core.Dispatcher.EngineDispatcher
-import cromwell.webservice.WorkflowJsonSupport._
-import cromwell.webservice.{APIResponse, CromwellApiService, SwaggerService}
-import cromwell.webservice.SprayCanHttpService._
+import cromwell.webservice.{AkkaHttpService, SwaggerService}
 import cromwell.webservice.WrappedRoute._
 import net.ceedubs.ficus.Ficus._
-import spray.http._
-import spray.json._
-import spray.routing.Route
 
 import scala.concurrent.duration._
-import scala.concurrent.{Await, ExecutionContextExecutor, Future}
+import scala.concurrent.{Await, Future}
 import scala.util.{Failure, Success}
 
 // Note that as per the language specification, this is instantiated lazily and only used when necessary (i.e. server mode)
 object CromwellServer {
-
   def run(cromwellSystem: CromwellSystem): Future[Any] = {
-    implicit val executionContext = scala.concurrent.ExecutionContext.Implicits.global
-
-    val actorSystem: ActorSystem = cromwellSystem.actorSystem
-    implicit val materializer: ActorMaterializer = cromwellSystem.materializer
-
-    val service = actorSystem.actorOf(CromwellServerActor.props(cromwellSystem.conf), "cromwell-service")
-    val webserviceConf = cromwellSystem.conf.getConfig("webservice")
-
-    val interface = webserviceConf.getString("interface")
-    val port = webserviceConf.getInt("port")
-    val timeout = webserviceConf.as[FiniteDuration]("binding-timeout")
-    val futureBind = service.bind(interface, port)(implicitly, timeout, actorSystem, implicitly)
-    futureBind andThen {
-      case Success(_) =>
-        actorSystem.log.info("Cromwell service started...")
-        Await.result(actorSystem.whenTerminated, Duration.Inf)
-      case Failure(throwable) =>
-        /*
-        TODO:
-        If/when CromwellServer behaves like a better async citizen, we may be less paranoid about our async log messages
-        not appearing due to the actor system shutdown. For now, synchronously print to the stderr so that the user has
-        some idea of why the server failed to start up.
-         */
-        Console.err.println(s"Binding failed interface $interface port $port")
-        throwable.printStackTrace(Console.err)
-        cromwellSystem.shutdownActorSystem()
+    implicit val actorSystem = cromwellSystem.actorSystem
+    implicit val materializer = ActorMaterializer()
```

Comment:
I would love to claim this was a stroke of genius but instead the answer appears to involve a shruggie. Changing it.